src/cmd/ksh93/tests/jobs.exp

```diff
@@ -0,0 +1,71 @@
+# Interactive tests for jobs, bg, fg and related builtins
+set pid [spawn $ksh]
+expect_prompt
+
+# ======
+# Ctrl-Z should stop current process
+log_test_entry
+send "cat\r"
+sleep 0.5
+send [ctrl Z]
+expect -re ".*Stopped.*cat\r\n" {
```

Comment:
This is failing sometimes on my macOS systems and on Travis macOS. The "stopped" message isn't being written. But it works reliably when I do this by hand. We'll probably need to blacklist this on macOS.

Reply:
This fails on macOS because the  interrupts cat's `read()` on stdin and it doesn't retry. Which results in the `cat` dying. You can work around that difference by using a fifo:

```
send "mkfifo fifo\r"
expect_prompt
send "cat < fifo\r"
```

Then change the pattern here and below from `cat\r\n` to `cat < fifo\r\n`. However, this also requires other changes. I recommend just blacklisting this entire unit test (in meson.build) on macOS with a "TODO" comment for now.